src/codegen/templates/property_getter_template.py.mako

```diff
@@ -1,13 +1,21 @@
 <%def name="script_property_getter(attribute)">\
 <%
         from codegen.utilities.text_wrappers import docstring_wrap
-        from codegen.utilities.attribute_helpers import get_generic_attribute_function_name, get_generic_attribute_function_type
+        from codegen.utilities.attribute_helpers import get_generic_attribute_function_name, get_generic_attribute_function_type, ATTRIBUTE_WITH_FILE_PATH_TYPE
     %>\
+    %if attribute.name in ATTRIBUTE_WITH_FILE_PATH_TYPE:
+    @property
+    def ${attribute.name}(self) -> Union[pathlib.Path, None]:
```

Comment:
```suggestion
    def ${attribute.name}(self) -> Optional[pathlib.Path]:
```

Reply:
Thanks for the suggestion, it's updated.